source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java

```diff
@@ -224,11 +226,16 @@ public static void setBackgroundImage(RenderedAdaptiveCard renderedCard,
         }
     }
 
-    public static void setSelectAction(RenderedAdaptiveCard renderedCard, BaseActionElement selectAction, View view, ICardActionHandler cardActionHandler)
+    public static void setSelectAction(RenderedAdaptiveCard renderedCard, BaseActionElement selectAction, View view, ICardActionHandler cardActionHandler, RenderArgs renderArgs)
```

Comment:
Can we add a unit test for the public API so we have automated test coverage for this scenario going forward?

Reply:
I added a unit test that covers the API